tests/anumargak_test.js

```diff
@@ -16,6 +16,21 @@ describe("Anumargak ", function () {
         expect(anumargak.quickFind("HEAD", "/this/is/static").handler()).toEqual(30);
     });
 
+    it("should register for multiple urls", function () {
+        var anumargak = Anumargak();
+
+        anumargak.on(["GET", "HEAD"], ["/500", "/502"], () => 30);
```

Comment:
Can you please add one more test to test direct methods `anumargak.get([ "url1", "url2"], () => 30 );`

Reply:
Sure I can do that, Sorry for the late reply.